compose/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
```

Comment:
This should live on `cardano-wallet` thought, don't you think ?

Reply:
OK.